src/chatlog/chatlog.cpp

```diff
@@ -195,7 +202,10 @@ void ChatLog::mouseReleaseEvent(QMouseEvent* ev)
 {
     QGraphicsView::mouseReleaseEvent(ev);
 
+    viewport()->unsetCursor();
     selectionScrollDir = NoDirection;
+
+    mouseMoveEvent(ev); // To fix the cursor.
```

Comment:
What is this good for? It "feels" wrong to call the `mouseMoveEvent` here. A separate function, which is called from both `mouseReleaseEvent` and `mouseMoveEvent` is better, don't you think?